scripts/presubmit.sh

```diff
@@ -74,6 +74,14 @@ main() {
     grep -v _string.go | \
     tr '\n' ' ')"
 
+  # Prevent the creation of proto files with .txt extensions.
+  bad_protos="$(find . -name '*.pb.txt' -o -name '*.proto.txt')"
+  if [[ "${#bad_protos}" -ne 0 ]]; then
+    echo "Text based protos must use the textproto extension:"
```

Comment:
Just to make it obvious that ".textproto" *is* the textproto extension...

```suggestion
    echo "Text-based protos must use the .textproto extension:"
```